source/macos/AdaptiveCards/AdaptiveCards/AdaptiveCard.swift

```diff
@@ -155,8 +155,9 @@ public struct InputFieldConfig {
     public let backgroundColor: NSColor
     public let borderColor: NSColor
     public let activeBorderColor: NSColor
+    public let placeholderTextColor: NSColor?
     
```

Comment:
```suggestion
    public let placeholderTextColor: NSColor
    
```
Can be non-optional. It's good to be optional, but let's maintain uniformity with other properties.